src/Compilers/Extension/CompilerPackage.cs

```diff
@@ -100,9 +95,14 @@ private void WriteMSBuildFiles(string packagePath, string hiveName)
             }
             catch (Exception e)
             {
+                var msg = 
+$@"{e.Message}
+
+To reload the Roslyn compiler package, close Visual Studio and any MSBuild processes, then restart Visual Studio.";
+
```

Comment:
@jasonmalinowski is there a reason this string isn't localized?  If not I'll open a bug.